lstchain/mc/sensitivity.py

```diff
@@ -352,8 +361,8 @@ def find_best_cuts_sensitivity(simtelfile_gammas, simtelfile_protons,
     mc_par_p['emin'] = mc_par_p['emin'].to(u.TeV)
     mc_par_p['emax'] = mc_par_p['emax'].to(u.TeV)
 
-    mc_par_g['area_sim'] = mc_par_g['area_sim'].to(u.cm ** 2)
-    mc_par_p['area_sim'] = mc_par_p['area_sim'].to(u.cm ** 2)
+    mc_par_g['area_sim'] = mc_par_g['area_sim'].to(u.m ** 2)
+    mc_par_p['area_sim'] = mc_par_p['area_sim'].to(u.m ** 2)
```

Comment:
why this change of units here?

Reply:
There was a reason but I forget why I changed them. I will go back to the previous version with cm.